### DISCUSSION TITLE
How to get Heart On Your Sleeve Achievement

### DISCUSSION THREAD

**Discussion by HarshanaJayasekara** [community/community#116681] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body
![image](https://github.com/community/community/assets/98016320/c34783a8-58b3-48e4-beae-c746ec7d77d9)

React to something on GitHub with a ❤️ emoji (Being tested)

**Comment by HarshanaJayasekara** [comment]
❤️

**Comment by queenofcorgis** [comment]
Hi @HarshanaJayasekara  

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
